src/inspector/view/qml/MuseScore/Inspector/notation/tempos/TempoSettings.qml

```diff
@@ -58,7 +58,7 @@ Column {
         propertyItem: root.model ? root.model.tempo : null
         enabled: root.model ? !root.model.isEmpty && !followWrittenTempoCheckbox.checked : false
 
-        measureUnitsSymbol: qsTrc("inspector", "BPM")
+        measureUnitsSymbol: qsTrc("inspector", " BPM")
```

Comment:
Translatable strings should not contain leading or trailing whitespace, because translators have no reasonable way to notice that. Generally this is solved by putting the extra space outside the translatable string:
```suggestion
        measureUnitsSymbol: " " + qsTrc("inspector", "BPM")
```

Reply:
Other than that we've never separated the numbers from the units by a space, but there had been pretty lengthy discussion about this.